libsawtooth/src/client/rest.rs

```diff
@@ -271,6 +304,34 @@ impl Into<ClientTransactionHeader> for TransactionHeader {
     }
 }
 
+impl Into<ClientBlock> for Block {
+    fn into(self) -> ClientBlock {
+        let mut clientbatches = Vec::new();
+        for b in self.batches {
+            let batch: ClientBatch = b.into();
+            clientbatches.push(batch);
+        }
```

Comment:
This can be done a bit more succinctly with `let clientbatches = self.batches.into_iter().map(ClientBatch::from)`

Reply:
Changed, had to do it slightly differently to get it to work